test/Makefile

```diff
@@ -4,6 +4,9 @@ TEST_HARNESS_PATH = $(realpath .test-harness)
 BATS_ARGS ?= --tap
 BATS_LOG ?= test.log
 
+# Expose GitHub Token as github_webhooks_token variable so that SSM does not fail
+export TF_VAR_github_webhooks_token ?= $(GITHUB_TOKEN)
```

Comment:
Oh, this is the wrong `Makefile`. Move it to `test/src/Makefile`

Reply:
(this is the `Makefile` for the `bats` automated testing. The tests are failing with `terratest`)

The `bats` is currently failing due to listing. run `terraform fmt` for that.

Reply:
I see ok